lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -35,6 +35,18 @@
 #include <pxr/base/vt/value.h>
 #include <pxr/imaging/glf/contextCaps.h>
 #include <pxr/imaging/hd/rprim.h>
+
+#include <hdMaya/delegates/delegateRegistry.h>
+#include <hdMaya/delegates/sceneDelegate.h>
+#include <hdMaya/utils.h>
+
+#include <mayaUsd/render/px_vp20/utils.h>
```

Comment:
I think we're saying these should be one group?

Reply:
Please see b6cb901eb